zipkin-server/pom.xml

```diff
@@ -609,6 +609,14 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>9</source>
```

Comment:
zipkin-server should stay 1.8, as directed from the parent pom ...

Reply:
sorry for pushing it by mistake